### ISSUE TITLE
Confusing / Unnecessary Code

### ISSUE THREAD

**Issue by chrisyeh96** [nishithbsk/tensorflow_tutorials#1]
On line 55 of the [CS 224N linear regression example](https://github.com/nishithbsk/tensorflow_tutorials/blob/master/cs224n/linear_regression/linear_regression_complete.py#L55), `loss_val` is a scalar, so there shouldn't be any reason to call `.mean()` on it. It is a scalar because no `axis` parameter is given in the `loss = tf.reduce_mean()` call on line 38.
